File: content/snippet/code-signal/arcade/intro/exploring-the-water/array-change/array-change.py
def array_change(inputArray):
    count = 0
    for i in range(1, len(inputArray)):
        if inputArray[i] <= inputArray[i - 1]:
            count += inputArray[i - 1] - inputArray[i] + 1
            inputArray[i] = inputArray[i - 1] + 1

    return count


print(array_change([-1000, 0, -2, 0]))

File: content/snippet/code-signal/arcade/intro/exploring-the-water/array-change/array-change.js
function arrayChange(inputArray) {
  let count = 0;
  for (let i = 0; i < inputArray.length; i++) {
    if (inputArray[i] >= inputArray[i + 1]) {
      const diff = inputArray[i] - inputArray[i + 1] + 1;
      inputArray[i + 1] += diff;
      count += diff;
    }
  }
  return count;
}

console.log(arrayChange([-1000, 0, -2, 0]));
